src/clr-angular/popover/signpost/signpost-trigger.ts

```diff
@@ -19,16 +30,30 @@ import { IfOpenService } from '../../utils/conditional/if-open.service';
  */
 export class ClrSignpostTrigger implements OnDestroy {
   private subscriptions: Subscription[] = [];
+  public ariaExpanded;
```

Comment:
Does `ariaExpanded` have a type?

It looks like it's a boolean...

Reply:
I thought it would get inferred by the `tsc` but will add it for Clarity.